$NetBSD: patch-ac,v 1.6 2012/07/30 02:53:57 dholland Exp $

- Add image libraries to link. (is this needed?)
- Honor LDFLAGS so rpaths work.
- Configure paths for pkgsrc.
- Use optimization level that actually exists (gcc) or works (clang).

--- fd2ps/Imakefile.orig	2002-10-01 16:40:02.000000000 +0000
+++ fd2ps/Imakefile
@@ -32,7 +32,7 @@ OBJS	= $(SRCS:.c=.o)
 #if !defined( PpcDarwinArchitecture )
 FORMSLIBDIR     = $(TOP)/lib
 FORMSINCDIR     = $(FORMSLIBDIR)       
-FORMSLIB        = -L$(FORMSLIBDIR) -lforms
+FORMSLIB        = -L$(FORMSLIBDIR) -lforms $(JPEGLIB) $(XPMLIB) $(LDFLAGS)
 DEPFORMSLIB     = $(FORMSLIBDIR)/libforms.a
 #endif
 
@@ -64,7 +64,7 @@ EXTRA_INCLUDES  = \
 		  -I.
 
 DEPEND_DEFINES	= \
-		  -I/usr/local/include
+		  -I${LOCALBASE}/include
 
 EXTRA_LIBRARIES	= \
 		  $(STATIC) \
@@ -83,7 +83,7 @@ PSRCS	= $(SPLSRCS) \
 CDEBUGFLAGS	= -g $(OPTIMIZER)
 #if Optimize
 IMAKE_OPTIMIZE  = -DDoOptimize=1
-OPTIMIZER	= -O4
+OPTIMIZER	= -O2
 #else
 OPTIMIZER	=
 #endif
